dotCMS/src/main/java/com/dotmarketing/db/DBPropertiesDataSourceStrategy.java

```diff
@@ -70,7 +70,7 @@ public DataSource apply() {
 
             properties.clear();
             return new HikariDataSource(config);
-        } catch (ConfigurationException | FileNotFoundException e) {
+        } catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5442148)